extensions/podman/packages/extension/src/extension.ts

```diff
@@ -185,6 +185,7 @@ function notifySetupPodman(): void {
 export async function updateMachines(
   provider: extensionApi.Provider,
   podmanConfiguration: PodmanConfiguration,
+  checkDefaultConnection: boolean,
```

Comment:
question: make it optional and default to false ?

so we would only one change in the codebase where we set true

Reply:
I am on old Java developer 🥲 

Should be good now